### PR TITLE
docs: add apisix casbin blog

### PR DESCRIPTION
**Pull request by SylviaBABY**
Changes:

add APISIX Casbin tech blog

### CODE REVIEW

File: website/blog/2021-08-23-Apache APISIX-Casbin.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+title: "在 Apache APISIX 中使用 Casbin 进行授权"
+keywords: 
+- Apache APISIX
+- Casbin
+- 访问控制
+- 插件,
```
Comment (yzeng25) [inline website/blog/2021-08-23-Apache APISIX-Casbin.md]: please delete comma

**Review by yzeng25 — APPROVED** [review]

File: website/blog/2021-08-23-Apache APISIX-Casbin.md (hunk 1 above)
Comment (SylviaBABY) [reply website/blog/2021-08-23-Apache APISIX-Casbin.md]: done

**Review by bzp2010 — APPROVED** [review]

File: website/blog/2021-08-23-Apache APISIX-Casbin.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
+---
+title: "在 Apache APISIX 中使用 Casbin 进行授权"
+keywords: 
+- Apache APISIX
+- Casbin
```
Comment (liuxiran) [inline website/blog/2021-08-23-Apache APISIX-Casbin.md]: `Cashbin` and `插件` = `Cashbin 插件` ?

File: website/blog/2021-08-23-Apache APISIX-Casbin.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    "uri": "/*"
+}'
+```
+
+### 利用全局模型/策略使用插件
+
+在一些情形中，你可能想要在多个路由中使用相同的模型和策略，你可以首先发送一个 PUT 请求将模型和策略的配置发送到插件的元数据：
+
+```shell
+curl <http://127.0.0.1:9080/apisix/admin/plugin_metadata/authz-casbin> -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -i -X PUT -d '
```
Comment (liuxiran) [inline website/blog/2021-08-23-Apache APISIX-Casbin.md]:
```suggestion
curl http://127.0.0.1:9080/apisix/admin/plugin_metadata/authz-casbin -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -i -X PUT -d '
```

File: website/blog/2021-08-23-Apache APISIX-Casbin.md (hunk 2 above)
Comment (bzp2010) [reply website/blog/2021-08-23-Apache APISIX-Casbin.md]: emm, i think remove "插件" better

Comment (SylviaBABY) [reply website/blog/2021-08-23-Apache APISIX-Casbin.md]: done, change into ➡️ authz-casbin

**Review by liuxiran — APPROVED** [review]

**Pull request merged**